pipettes/core/CMakeLists.txt

```diff
@@ -15,6 +13,7 @@ function(target_pipettes_core_common TARGET REVISION)
 endfunction()
 
 function(target_pipettes_core_single TARGET REVISION)
+    target_compile_definitions(${TARGET} PUBLIC USE_SENSOR_MOVE)
```

Comment:
I don't think we need this here, I think the def in target_pipettes_core_common will be sufficient